libminifi/src/core/repository/VolatileContentRepository.cpp

```diff
@@ -45,10 +48,10 @@ bool VolatileContentRepository::initialize(const std::shared_ptr<Configure> &con
     }
   }
   if (!minimize_locking_) {
-    for (auto ent : value_vector_) {
+    for (auto ent : repo_data_.value_vector) {
       delete ent;
     }
-    value_vector_.clear();
+    repo_data_.value_vector.clear();
```

Comment:
I would move this to `VolatileRepositoryData` as a new `clear()` function.

Reply:
Updated in 79d5a6e5d2d061953ebc87b82d8cc10da6bf10d6